libs/core/src/lib/utils/functions/scroll.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Function used to scroll specified element by some distance
+ *
+ * @param containerElement     - Container element scrolled
+ * @param distanceToScroll     - Distance of scroll in px
+ * */
+export function scrollTop(containerElement: Element, distanceToScroll: number): void {
```

Comment:
Shouldn't be named as native alternative `scrollTo`?

Reply:
I'm not sure, because there is used scrollTo with top parameter, or scrollTop. I wanted to leave space for scrollLeft method